taipy/core/taipy.py

```diff
@@ -1015,12 +1015,11 @@ def import_scenario(folder_path: Union[str, pathlib.Path], override: bool = Fals
         override (bool): If True, override the entities if existed. Default value is False.
 
     Return:
-        The imported scenario.
+        The imported scenario if the import is successful.
```

Comment:
I believe it is not necessary to add that. It is implicit for me.

Reply:
We can discuss this